Exercises-4-OOP/src/inheritanceanimal/Dog.java

```diff
@@ -0,0 +1,29 @@
+package inheritanceanimal;
+
+public class Dog extends Mammal {
+
+	public Dog() {
```

Comment:
This constructor is not part of the requirements, should be removed.

Reply:
if I remove it, the application won't run :(